src/Datadog.Trace/HttpOverStreams/DatadogHttpHeaderHelper.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Datadog.Trace.HttpOverStreams
+{
+    internal class DatadogHttpHeaderHelper
+    {
+        public const char CarriageReturn = '\r';
+        public const char LineFeed = '\n';
+        public static readonly string CrLf = Environment.NewLine;
```

Comment:
`Environment.NewLine` depends of the platform, on [windows-.NET Framework](https://referencesource.microsoft.com/#mscorlib/system/environment.cs,1067) is `\r\n` (CRLF) . but in linux the constant is `\n` (LF). So the name of the constant is not accurate and also I don't know if we want this to be different for an HttpHeader or if the Http RFC makes clear this difference.

Reply:
I renamed these for now.
This will not be used in linux environments at current time, and I think that testing could be in a separate effort.